test/e2e/erc721/erc721DifferentChainsMock.js

```diff
@@ -0,0 +1,226 @@
+const TruffleAssert = require('truffle-assertions');
+const Ethers = require('ethers');
+
+const RelayerContract = artifacts.require("Relayer");
+const BridgeContract = artifacts.require("Bridge");
+const ERC721MintableContract = artifacts.require("ERC721Mintable");
+const ERC721HandlerContract = artifacts.require("ERC721Handler");
+
+contract('E2E ERC721 - Two EVM Chains', async accounts => {
```

Comment:
How do you actually test two chains? Where is the second chain spun up?

Reply:
It's not actually two chains, just using two separate contracts and two different `chainID`s to mock two EVM chains